java-checks-test-sources/src/main/java/checks/NoTestsInProductionFile.java

```diff
@@ -0,0 +1,9 @@
+package checks;
+
+import org.junit.jupiter.api.Test;
+
+public class NoTestsInProductionFile { // Noncompliant [[sc=14;ec=35]] {{Move this test class to a separate assembly in this solution.}}
```

Comment:
Remove this `Noncompliant` message, and replace it by `compliant`, as we do not expect any issue here. I guess it's a copy-paste issue, but this is misleading.